src/foxlator_lib/utils.py

```diff
@@ -1,4 +1,9 @@
 
-def get_version():
+def get_version() -> str:
     import importlib.metadata
     return importlib.metadata.version('foxlator_lib')
+
+
+class FileSystem(object):
+    def is_file(self) -> bool:
+        pass
```

Comment:
I would move this to another file due to the fact that this class can grow

Reply:
Yep, I totally agree. I'll create separated directory for all utilities.